### ISSUE TITLE
Error: fromBuildId or toBuildId is null. Supply a valid value and try again.

### ISSUE THREAD

**Issue by andreasglaubitz** [rfennell/AzurePipelines#1035]
### Azure DevOps Extensions

Generate Release Notes (Node Cross Platform)

### Platform

Azure DevOps Services

### Extension Version

3.55.3

### Describe the bug

```bash
Since update from version 3.54.4 to 3.55.3 the task will fail with a warning. 
I get the error message: 


{ Error: fromBuildId or toBuildId is null. Supply a valid value and try again.
Total Builds: [0]
    at RestClient.<anonymous> (D:\a\_tasks\XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb\3.55.3\node_modules\typed-rest-client\RestClient.js:202:31)
    at Generator.next (<anonymous>)
    at fulfilled (D:\a\_tasks\XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb\3.55.3\node_modules\typed-rest-client\RestClient.js:6:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  statusCode: 400,
  result:
   { '$id': '1',
     innerException: null,
     message:
      'fromBuildId or toBuildId is null. Supply a valid value and try again.',
     typeName:
      'Microsoft.TeamFoundation.Build.WebApi.InvalidBuildQueryException, Microsoft.TeamFoundation.Build2.WebApi',
     typeKey: 'InvalidBuildQueryException',
     errorCode: 0,
     eventId: 3000 } }
```

It seems the output is OK. But the task fails with a warning.
```


### Repo Steps

```bash
I use the task inside a yaml build pipeline with options:
checkstage=true 
overrideBuildReleaseId

Any ideas for the reason?
```

### Expected Behavior

```bash
Task finishes without warning
```

**Comment by rfennell** [comment]
Without your log and I cannot be sure, but I think the issue is a warning because it has tried to look for associated work items and commits for a consumed artefact that is not of the type `pipeline` when the `checkstage=true` flag is set.

I have just added an extra code path for this case and it is running through my release process now. It will be released as 3.56.x a an hour or two

**Comment by andreasglaubitz** [comment]
Thank you Richard, 

your change fixed the problem! 😄
